### PR TITLE
James 1988 Typos

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

**Comment by chibenwa** [comment]
You intoduced error:

```
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] getMailetShouldLoadClassWhenInExtensionsJars(org.apache.james.utils.GuiceMailetLoaderTest)  Time elapsed: 0.005 sec  <<< ERROR!
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] java.lang.NullPointerException
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 	at org.apache.james.utils.GuiceMailetLoaderTest.getMailetShouldLoadClassWhenInExtensionsJars(GuiceMailetLoaderTest.java:74)
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] getMailedShouldAllowCustomPackages(org.apache.james.utils.GuiceMailetLoaderTest)  Time elapsed: 0.001 sec  <<< ERROR!
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] java.lang.NullPointerException
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 	at org.apache.james.utils.GuiceMailetLoaderTest.getMailedShouldAllowCustomPackages(GuiceMailetLoaderTest.java:102)
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] getMailetShouldThrowOnUnknownMailet(org.apache.james.utils.GuiceMailetLoaderTest)  Time elapsed: 0.001 sec  <<< ERROR!
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] java.lang.NullPointerException
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 	at org.apache.james.utils.GuiceMailetLoaderTest.getMailetShouldThrowOnUnknownMailet(GuiceMailetLoaderTest.java:116)
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] Running org.apache.james.utils.GuiceMatcherLoaderTest
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] [main] INFO org.apache.james.utils.ExtendedClassLoader - Loading custom classpath resource /james-project/server/container/guice/mailet/target/test-classes/recursive/included-jars/james-server-guice-custom-mailets-3.0.0-beta6-SNAPSHOT.jar
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] [main] INFO org.apache.james.utils.ExtendedClassLoader - No extensions-jars/ folder.
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] [main] INFO org.apache.james.utils.ExtendedClassLoader - Loading custom classpath resource /james-project/server/container/guice/mailet/target/test-classes/recursive/included-jars/james-server-guice-custom-mailets-3.0.0-beta6-SNAPSHOT.jar
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] [main] INFO org.apache.james.utils.ExtendedClassLoader - No extensions-jars/ folder.
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] Tests run: 6, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.268 sec <<< FAILURE! - in org.apache.james.utils.GuiceMatcherLoaderTest
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] getMatcherShouldThrowOnUnknownMailet(org.apache.james.utils.GuiceMatcherLoaderTest)  Time elapsed: 0.004 sec  <<< ERROR!
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] java.lang.NullPointerException
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 	at org.apache.james.utils.GuiceMatcherLoaderTest.getMatcherShouldThrowOnUnknownMailet(GuiceMatcherLoaderTest.java:116)
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] getMatcherShouldLoadClassWhenInExtensionsJars(org.apache.james.utils.GuiceMatcherLoaderTest)  Time elapsed: 0 sec  <<< ERROR!
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] java.lang.NullPointerException
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 	at org.apache.james.utils.GuiceMatcherLoaderTest.getMatcherShouldLoadClassWhenInExtensionsJars(GuiceMatcherLoaderTest.java:74)
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] Results :
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4] Tests in error: 
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4]   GuiceMailetLoaderTest.getMailedShouldAllowCustomPackages:102 » NullPointer
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4]   GuiceMailetLoaderTest.getMailetShouldLoadClassWhenInExtensionsJars:74 » NullPointer
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4]   GuiceMailetLoaderTest.getMailetShouldThrowOnUnknownMailet:116 » NullPointer
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4]   GuiceMatcherLoaderTest.getMatcherShouldLoadClassWhenInExtensionsJars:74 » NullPointer
[Building Java 8 james-1988-df435cb4-f522-4b4c-ac30-70687b7b1ad4]   GuiceMatcherLoaderTest.getMatcherShouldThrowOnUnknownMailet:116 » NullPointer
```

File: server/container/guice/mailet/src/test/java/org/apache/james/utils/FileSystemFixture.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public InputStream getResource(String url) throws IOException {
 
         @Override
         public File getFile(String fileURL) throws FileNotFoundException {
-            return new File(ClassLoader.getSystemResource("recursive/included-jars").getFile());
```
Comment (chibenwa) [inline server/container/guice/mailet/src/test/java/org/apache/james/utils/FileSystemFixture.java]: I guess the NPE comes from not having renamed the test resources folder to **recursive/extensions-jars**

Comment (rouazana) [inline server/container/guice/mailet/src/test/java/org/apache/james/utils/FileSystemFixture.java]: yep sorry, fixed

**Comment by chibenwa** [comment]
Just merged

**Pull request closed without merging**